ir-sanguinem-et-lacrimis/common/units/equipment/anti_air.txt

```diff
@@ -0,0 +1,82 @@
+equipments = {
+
+	anti_air_equipment = {
+		is_archetype = yes
+		picture = archetype_anti_air_equipment
+		type = {
+			anti_air
+			infantry #adding infantry type
+		}
+		group_by = archetype
+
+		interface_category = interface_category_land
+
+		#speed
+		maximum_speed = 4
```

Comment:
the speed of units is set by their equipment. If the equipment has no speed set, then it's going to be the slowest speed, which in turn is set by the defines